gsplat/cuda/_wrapper.py

```diff
@@ -77,12 +77,13 @@ def quat_scale_to_covar_preci(
     return covars if compute_covar else None, precis if compute_preci else None
 
 
-def persp_proj(
+def proj(
```

Comment:
For the sake of backward compatibility, I think we should keep the function `persp_proj` and have another `ortho_proj`. The rest of the code does not need to change (i.e., these two APIs could share the underlying implementation)

Another solution which is also fine with me, is to have this `proj` function and keep the original `persp_proj` but raise deprecated warning inside of it. This would also avoid sudden interrupt to the users.